inc/Nav_Menus/Component.php

```diff
@@ -59,15 +67,55 @@ public function template_tags() : array {
 		);
 	}
 
+	/**
+	 * Set the navs menus slugs and their description.
+	 *
+	 * @return void
+	 */
+	public function set_navs_menus() {
+		$this->nav_menus = array(
+			'primary' => esc_html__( 'Primary', 'wp-rig' ),
+		);
+	}
+
+	/**
+	 * Get the navs menus slugs and their description.
+	 *
+	 * @return array Associative array of menu location identifiers (like a slug) and descriptive text.
+	 */
+	public function get_navs_menus() : array {
+		return $this->nav_menus;
+	}
+
+	/**
+	 * Get the navs menus slugs.
+	 *
+	 * @return array Array of menu location identifiers (like a slug).
+	 */
+	public function get_menus_slugs() : array {
+		return array_keys( $this->nav_menus );
+	}
+
+	/**
+	 * Get the primary nav menu slug.
+	 *
+	 * @return string Primary menu location identifier (like a slug).
+	 */
+	public function get_primary_menu_slug() : string {
```

Comment:
Let's remove this and stick with the constant as "always" on. That should be enforced for consistency.